### DISCUSSION TITLE
SealedSecret metadata.annotations too long (not able to apply)

### DISCUSSION THREAD

**Discussion by goseind** [bitnami-labs/sealed-secrets#1048] [Q&A] [answered]
I created a SealedSecret containing roughly ~500 certificates, but when I want to apply it, the following error occurs:

`The SealedSecret "test" is invalid: metadata.annotations: Too long: must have at most 262144 bytes`

I'm not sure whether this is an issue, so I wanted to see if anyone has attempted something like this before and had the same problems.

Thanks in advance for sharing your experience!

**Comment by chinmaya-n** [comment, marked as answer]
When you or your CD tool deploys to Kubernetes, use the `--server-side` option with `kubectl apply`. [More info](https://kubernetes.io/docs/reference/using-api/server-side-apply/). The reason for this is the `last-applied` annotation gets too long due to the size of the large manifest.
